packages/vitest/src/node/config.ts

```diff
@@ -56,6 +56,16 @@ export function resolveApiConfig<Options extends ApiConfig & UserConfig>(
   return api
 }
 
+const configError = (error: string): never => {
+  console.warn(
+    c.yellow(
+      `${c.inverse(c.red(' VITEST '))} ${error}\n`,
+    ),
+  )
+
+  process.exit(1)
```

Comment:
Better to just throw the error, as `resolveConfig` can be used dynamically, and the caller might want to catch the error.

Reply:
We can catch it and process.exit in the caller, instead of having side-effect for `resolveConfig`

Reply:
I've added throwing, but I still do `process.exit` when catching the error